frontend/src/i18n/locales/de.yaml

```diff
@@ -1 +1,4 @@
+language-name: Deutsch
```

Comment:
Mhhh good question. I don't think I care. kebap-case might be a bit easier to read, but yeah dunno. Do you have an opinion?

Reply:
I prefer `kebab-case` to `camelCase`. In general, I prefer `snake_case` even more, but I don't know if I would use it here, because of the context thing, even if we can make it work. `bare words` are cool as well, but it looks a bit weird if you have `nested.bare words.in there`, since the nesting separator binds more strongly than the space.

Reply:
I also just learned that we can configure all these separators. We could have `snake_case/plural` kebab-case:context`, `nesting/with/slashes` or whatever else we want. xD

I personally like the first one, but I guess no one really cares that much, and sticking to the defaults might just be the best solution, in which case I would probably either use `kebab-case` or `bare words`, the latter especially if we don't plan on using nested keys.

Reply:
> sticking to the defaults might just be the best solution

Agreed

>  if we don't plan on using nested keys.

I think we will. So `kebap-case` it is. As that's already what this PR does, I will resolve this discussion.